### PR TITLE
fix(solana_pusher): forgot await

### PR DESCRIPTION
**Pull request by guibescos**

### CODE REVIEW

File: price_pusher/src/solana/solana.ts [hunk 1]
```diff
@@ -78,13 +78,17 @@ export class SolanaPricePusher implements IPricePusher {
...
+      this.shardId
+    );
 
     try {
       await this.pythSolanaReceiver.provider.sendAll(
         await transactionBuilder.buildVersionedTransactions({
           computeUnitPriceMicroLamports: this.computeUnitPriceMicroLamports,
-        })
+        }),
+        { preflightCommitment: "processed" }
```
Comment (guibescos) [inline price_pusher/src/solana/solana.ts:91]: Update preflightCommitment to avoid simulation errors

Comment (ali-bahjati) [reply price_pusher/src/solana/solana.ts:91]: why not skip it entirely :?

**Review by ali-bahjati — APPROVED** [review]

**Review by ali-bahjati — APPROVED** [review]
maybe bump the version too?

File: price_pusher/src/solana/solana.ts (hunk 1 above)
Comment (guibescos) [reply price_pusher/src/solana/solana.ts:91]: Ok

**Pull request merged**